tests/unit/pmodel/test_optimal_chi.py

```diff
@@ -0,0 +1,182 @@
+"""Testing Optimal submodule."""
+
+
+import numpy as np
+import pytest
+
+from pyrealm.pmodel.optimal_chi import (
+    OptimalChiC4,
+    OptimalChiC4NoGamma,
+    OptimalChiC4NoGammaRootzoneStress,
+    OptimalChiC4RootzoneStress,
+    OptimalChiLavergne20C3,
+    OptimalChiLavergne20C4,
+    OptimalChiPrentice14,
+    OptimalChiPrentice14RootzoneStress,
+)
+from pyrealm.pmodel.pmodel_environment import PModelEnvironment
```

Comment:
In general, I've a preference for importing the `pyrealm` components within tests. I've been told this is better practice, because it isolates the import process within the test itself, so the import becomes part of the test and only one test fails if an import fails. @MarionBWeinzierl and @mondus, is that right?

Having said that, this is _wildly_ inconsistent in the code base at the moment. I learned this part way through writing the existing `pyrealm` tests 😄 , so it might be better just to close this PR as is and then have a new issue to regularise the import locations.

Reply:
Yeah, makes sense. I will create a new issue to include imports within tests that are using these imports in all test files.